enterprise-ws-spec/src/main/asciidoc/ServerProgrammingModel.adoc

```diff
@@ -650,32 +528,31 @@ is called.
 
```

Comment:
same here - this can be moved one level up

Reply:
I can't find where this is, let me know if it's not fixed when you review my updates.

Reply:
it's ok now, probably fixed by other changes